### PR TITLE
clientv3: translate WithPrefix() into WithFromKey() for empty key

### PR DESCRIPTION
**Pull request by heyitsanthony**
Users seem to expect the client to do this.

### CODE REVIEW

File: clientv3/integration/kv_test.go [hunk 1]
```diff
@@ -381,6 +381,36 @@ func TestKVRange(t *testing.T) {
 				{Key: []byte("fop"), Value: nil, CreateRevision: 9, ModRevision: 9, Version: 1},
 			},
 		},
+		// fetch entire keyspace using WithFromKey
+		{
+			"\x00", "",
```
Comment (fanminshi) [inline clientv3/integration/kv_test.go]: if the key is `""` with the opt `WithFromKey()`, what's the expected behavior?

Comment (heyitsanthony) [inline clientv3/integration/kv_test.go]: It'll fail because no key is defined. No one complains about Get(ctx, "", WithFromKey())

Comment (fanminshi) [inline clientv3/integration/kv_test.go]: got it.

**Comment by fanminshi** [comment]
lgtm

**Comment by xiang90** [comment]
lgtm

**Pull request merged**